src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/PackageReference.xaml

```diff
@@ -13,6 +13,10 @@
                 SourceOfDefaultValue="AfterContext"
                 SourceType="TargetResults" />
   </Rule.DataSource>
+  
+  <StringProperty Name="Aliases"
+                  Description="Aliases for the assembly references coming from this PackageReference."
+                  DisplayName="Aliases" />
```

Comment:
Remove DisplayName, it doesn't need to be present if names matches.

Reply:
It does if you want it to be localised. We have a test that will fail if removed, IIRC.